demo-shell/src/app/components/files/files.component.html

```diff
@@ -653,6 +653,30 @@ <h1>You don't have permissions</h1>
             </mat-form-field>
         </form>
 
+        <section>
+            <mat-slide-toggle
+                color="primary" [(ngModel)]="enableDownloadPrompt" id="enableDownloadPrompt" (change)="onEnableNonResponsiveDialogChange()">
+                Enable Download Prompt
+            </mat-slide-toggle>
+        </section>
+        <section *ngIf="enableDownloadPrompt">
+            <mat-form-field>
+                <input matInput type="number" [(ngModel)]="downloadPromptDelay" (change)="onNonResponsiveInitialTimerChange()">
+            </mat-form-field>
+        </section>
+
+        <section *ngIf="enableDownloadPrompt">
+            <mat-slide-toggle
+                color="primary" [(ngModel)]="enableDownloadPromptReminders" id="enableDownloadPromptReminders" (change)="onEnableNonResponsiveDialogReminderChange()">
+                Enable DownloadPrompt Reminders
```

Comment:
I think you're missing whitespace between `Download` and `Prompt`

Reply:
Added missing white space. Thanks!!